data/swtpm/ssh_script

```diff
@@ -2,18 +2,22 @@
  
 # This script is used for access remote host via ssh and run some commands
 # without interactive operations
+# In this test, we add some basic test scenarios for swtpm
 # Usage: ssh_script [host] [user] [password]
 # Reference: https://www.journaldev.com/1405/expect-script-ssh-example-tutorial
 # Author: https://www.journaldev.com/author/pankaj
 
 set a_exp $::env(TPM_CHK_CMD)
-set timeout 60
+set timeout 100
 spawn ssh [lindex $argv 1]@[lindex $argv 0]
 expect "yes/no*" {
     send "yes\r"
     expect "*?assword" { send "[lindex $argv 2]\r" }
     } "*?assword" { send "[lindex $argv 2]\r" }
 
 expect " #" { send "ls /dev/tpm*\r" }
+expect " #" { send "ls /sys/kernel/security/tpm0/binary_bios_measurements\r" }
+expect " #" { send "cat /sys/kernel/security/tpm0/binary_bios_measurements\r" }
+expect " #" { send "tpm2_eventlog /sys/kernel/security/tpm0/binary_bios_measurements\r" }
```

Comment:
Could you please define a variable for "/sys/kernel/security/tpm0/binary_bios_measurements" ?

Reply:
I can